go.mod

```diff
@@ -44,14 +44,15 @@ require (
 )
 
 replace (
+	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.8.1
 	// Kubernetes v1.19.7
 	k8s.io/api => k8s.io/api v0.19.7
 	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.7
 	k8s.io/apimachinery => k8s.io/apimachinery v0.19.7
 	k8s.io/client-go => k8s.io/client-go v0.19.7
 	k8s.io/code-generator => k8s.io/code-generator v0.19.7
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
-	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20210212083459-1f8c3e444cf5
-	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20210406112945-bc5e764e21e7
-	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20210301102056-4e0f782522f1
+	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20210406160751-ea335983df85
+	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20210408170221-7f92a63cf408
```

Comment:
related, nor do I get this.....  and why 0.19  ? :-) 

syntax from hell!!! :smile:

Reply:
Hey, this is just theory but I think it takes the latest tag in that repo starting with `v` and adds the timestamp to it (don't ask me why:)
* openshift/knative-eventing - latest tag: https://github.com/openshift/knative-eventing/releases/tag/v0.99.0
* openshift/knative-serving - latest tag: https://github.com/openshift/knative-serving/releases/tag/v0.10.0
* openshift-knative/eventing-kafka - latest tag: https://github.com/openshift-knative/eventing-kafka/releases/tag/v0.19.1